packages/patternfly-4/react-integration/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@patternfly/react-integration",
+  "version": "1.0.0",
+  "description": "Integration testing for PF4 using demo applications",
+  "main": "lib/index.js",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/patternfly/patternfly-react.git"
+  },
+  "keywords": [
+    "react",
+    "patternfly"
+  ],
+  "author": "Red Hat",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/patternfly/patternfly-react/issues"
+  },
+  "homepage": "https://github.com/patternfly/patternfly-react#readme",
+  "scripts": {
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run",
+    "integration:setup": "rimraf node_modules && yarn install",
+    "integration:test": "node ./scripts/server.js"
+  },
+  "dependencies": {
+    "@patternfly/react-core": "file:../react-core",
```

Comment:
for local e2e runs and development, this makes perfect sense to me. This would essentially test this PR or the latest code in this commit (also good). I would just hope that as a followup eventually, there is some way to run against a released version or tag, i.e. `"@patternfly/react-core" : "next"` to search for regressions we might be targeting before doing a "latest" release of the same code (to leave a consumer unaffected until all components load and pass e2e for the release). Maybe this could be done in a different app/deployment? Nice work! 👍

Reply:
I didn't realize you could do this! (depend on `file:...` in package.json). That's pretty cool. @dlabaj you can ignore the first part of my comment above, I hadn't noticed this in my first pass.

Reply:
@mturley No worries a lot of errors are caught just trying to build the demo app for example:

Example of app using dist unable to build:
*******************************************

$ react-scripts build
Creating an optimized production build...
Failed to compile.

./src/App.tsx
Attempted import error: 'Avatar' is not exported from '@patternfly/react-core'.


info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build:demo stderr:
error Command failed with exit code 1.

lerna ERR! yarn run build:demo exited 1 in 'demo-app-ts'
error Command failed with exit code 1.

Reply:
@mturley Others are caught when running the tests for example say a property is named wrong in the d.ts file.

Reply:
@priley86 That's a good point about testing against other builds, wondering if we can over ride it.  Will look into it further.

Reply:
Nice, thanks for the info @dlabaj. This looks great!